### DISCUSSION TITLE
NexoPOS 4.8.17

### DISCUSSION THREAD

**Discussion by Blair2004** [Blair2004/NexoPOS#1420] [Releases]
Today we're releasing an update that comes with quite important changes. We've also added a so awaited tax breakdown on receipt (@mbugikp you have been heard). That tax breakdown is about to be merged on v5.x and implemented on each Nexo Print Adapter updates (4.8.7). The following screenshot explains how to enabled it, as it's disabled by default.

![tax breakdown](https://user-images.githubusercontent.com/5265663/231571835-c46ab363-b1c6-41fd-8400-c64664096ec2.jpg)

We'll details further the changes and finish by thanking the contributors.

## Changelog:
- Added: Fixing spanish localization (by @FastAct)
- Fixed: stateful domain omitting port.
- Fixed: changing the tax group wasn't reflecting.
- Added: tax breakdown on invoice

## Special Thanks
- @mbugikp
- @FastAct 
- All of you.

<hr /><em>This discussion was created from the release <a href='https://github.com/Blair2004/NexoPOS/releases/tag/v4.8.17'>NexoPOS 4.8.17</a>.</em>

**Comment by mbugikp** [comment]
Thank you for adding the tax breakdown on receipt/invoice.
